integrated_channels/degreed/exporters/content_metadata.py

```diff
@@ -78,13 +78,7 @@ def transform_image(self, content_metadata_item):
         """
         Return the image URI of the content item.
         """
-        image_url = ''
```

Comment:
@irfanuddinahmad I'm wondering why we previously has different transformation logic for `degreed` and `SAP SF`